backend/tracim_backend/lib/core/user.py

```diff
@@ -148,7 +148,7 @@ def get_all(self) -> typing.Iterable[User]:
 
     def get_user_ids_from_profile(self, profile: Profile) -> typing.Iterable[int]:
         query = self._apply_base_filters(self._session.query(User.user_id))
-        return query.filter(User.profile == profile).all()
+        return [res[0] for res in query.filter(User.profile == profile)]
```

Comment:
Not sure to understand the meaning of this change. Can you explain @grignards ?

Reply:
SQLAlchemy returns tuples, even when only one column is asked for (like it is here), so I extract the ids from their tuples.

Reply:
ok